test.properties

```diff
@@ -2260,7 +2260,9 @@
         **/site-navigation-menu-item-asset-vocabulary/**,\
         **/site-navigation-menu-item-display-page/**,\
         **/template/template-taglib/**,\
-        **/template/template-web/**
+        **/template/template-web/**,\
+        sdk/gradle-plugins-app-docker/**/*Test.java,\
+        sdk/gradle-plugins-test-integration/**/*Test.java
```

Comment:
Hey Nikki, can we change the patterns to just `sdk/**/gradleTest/**/*Test.java`? That should future-proof it and cover everything.